### PR TITLE
fix: none 0conf

### PR DESCRIPTION
**Pull request by ubavic**

### CODE REVIEW

**Comment by ubavic** [comment]
[Cash-out always resulting in a pending transaction](https://trello.com/c/SlzpsEzR/997-cash-out-always-resulting-in-a-pending-transaction)

**Review by siiky — APPROVED** [review]
LGTM

File: lib/wallet.js [hunk 1]
```diff
@@ -175,6 +175,8 @@ function authorizeZeroConf (settings, tx, machineId) {
 
   if (plugin === 'all-zero-conf') return Promise.resolve(true)
 
+  if (plugin === 'none') return Promise.resolve(false)
```
Comment (RafaelTaranto) [inline lib/wallet.js]: I don't think this is correct, if no zeroConf plugin is configured and tx.fiat < zeroConfLimit why would we wait for confirmation?

Also we don't use 'all-zero-conf' anymore, so I think this PR can be renaming 'all-zero-conf' to 'none'

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/wallet.js (hunk 1 above)
Comment (siiky) [reply lib/wallet.js]: Ah I misunderstood `tx.fiat.gt(zeroConfLimit)`... If there's no plugin, then the tx should be always accepted, no matter the limit, no? That's what I thought initially.

**Pull request merged**